eng/Packaging.props

```diff
@@ -1,4 +1,4 @@
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build">
```

Comment:
You can remove DefaultTargets="Build", especially from props/targets files.

Reply:
I removed that attribute from all props and targets file.